### ISSUE TITLE
Blank HN, console error

### ISSUE THREAD

**Issue by jjt** [etcet/HNES#77]
```
Uncaught TypeError: document.getElementById(...).firstChild.getAttribute is not a function
    HN.setTopColor @ hn.js:1289
    HN.rewriteUserNav @ hn.js:1186
    HN.init @ hn.js:407
    (anonymous function) @ hn.js:1464
```

**Comment by jjt** [comment]
![image](https://cloud.githubusercontent.com/assets/399469/7169890/722a2730-e37e-11e4-9b27-8c6d307f8272.png)

**Comment by lee-reinhardt** [comment]
Caused by this function:

```
	setTopColor: function(){
	  var topcolor = document.getElementById("header").firstChild.getAttribute("bgcolor");
	  if(topcolor.toLowerCase() != '#ff6600')
	  {
		document.getElementById("header").style.setProperty("background-color", topcolor, "important");
	  }
	},
```

There doesn't appear to be any elements on the page with the id `header`. Interestingly, I can't find a time (via http://archive.org/web/) where one ever did exist.

**Comment by hatboysam** [comment]
+1

**Comment by scttcper** [comment]
:+1:

**Comment by etcet** [comment]
I've merged bsloane1650's fix from #78 .
